flink-runtime/src/main/java/org/apache/flink/runtime/util/EnvironmentInformation.java

```diff
@@ -367,5 +478,13 @@ public RevisionInformation(String commitId, String commitDate) {
 			this.commitId = commitId;
 			this.commitDate = commitDate;
 		}
+
+		@Override
+		public String toString() {
```

Comment:
Why are we adding this?

Reply:
I added this because while debugging IntelliJ uses the toString method to show the content of a record ... which is messy if it falls back to the Object.toString(). I'll remove it.